exercises/high-scores/canonical-data.json

```diff
@@ -0,0 +1,91 @@
+{
+  "exercise": "high-scores",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Returns the list of scores",
+      "property": "scores",
+      "input": {
+        "scores": [3, 5, 2, 7]
+      },
+      "expected": [3, 5, 2, 7]
+    },
+    {
+      "description": "Returns the last score",
+      "property": "latest",
+      "input": {
+        "scores": [10, 0, 9, 3]
+      },
+      "expected": 3
+    },
+    {
+      "description": "Returns the highest score",
+      "property": "highest",
+      "input": {
+        "scores": [4, 10, 7]
+      },
+      "expected": 10
+    },
+    {
+      "description": "Returns the top 3 scores",
+      "cases": [
+        {
+          "description": "Returns personal bests",
+          "property": "top",
+          "input": {
+            "scores": [5, 3, 1]
+          },
+          "expected": [5, 3, 1]
+        },
+        {
+          "description": "Returns personal bests highest to lowest",
+          "property": "top",
+          "input": {
+            "scores": [2, 1, 3]
+          },
+          "expected": [3, 2, 1]
+        },
```

Comment:
Do we need a test with tied scores?

Reply:
Thanks for the review @Insti :+1:.

As for your question: maybe. I assume you're asking because without it those cases will be ambiguous for students, right? If that's the case, yes we can specify that :+1:

Reply:
Sounds good, thank you. FYI we're planning on returning `[4, 4, 3]` for `[4, 2, 4, 3]`. So yeah, it's a bit prescriptive to not using sets, but hopefully that's ok.

Reply:
This has now been added. Check "Returns personal bests when there is a tie".